workshops/website_mockup_generator/README.md

```diff
@@ -0,0 +1,284 @@
+
+[final_live_demo]: https://hcworkshop.19eucs071kumara.repl.co/
+[final_code]: https://repl.it/@19EUCS071KUMARA/Website-Mockup-Generator
+[repl_it]: https://repl.it
+
+## Part 1: Prerequisites
+
+It is recommended to have a basic understanding of:
+
+- HTML
+- JavaScript
+- CSS (totally optional for you to customize the look)
+
+However, this workshop is beginner friendly and you can refer the [final code][final_code] to understand better.
+
+
+## Part 2: Getting started
+
+### Setting up Repl.it
+
+[Repl.it](https://repl.it) is an amazing online code editor where we will be writing our code. Though repl.it is not mandatory, I strongly recommend you to use it in order to avoid installations and downloads.
+
+To get started, Create your Repl by going to [https://repl.it/languages/html](https://repl.it/languages/html). Your coding environment will be created in a few moments.
+
+![Setting up repl.it](https://cloud-flfptkrmk.vercel.app/0setuprepl.gif)
+
+Once your repl is setup, you are all set to continue!
+
+
+## Part 3: Building The Website Mockup Generator
+
+After your repl is setup successfully, you will be able to find 3 files on the left pane :
+- index.html
+- script.js
+- style.css
+
+We will be working with all the three files in this workshop. So, get your hands on the keyboard for some code!
+
+![Coding cat](https://media.giphy.com/media/LmNwrBhejkK9EFP504/giphy.gif)
+
+### 1) index.html
+
+First, let us write the required HTML code for the website mockup generator. You will be having the following code in `index.html` by default :
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>repl.it</title>
+    <link href="style.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body>
+    <script src="script.js"></script>
+  </body>
+</html>
+```
+#### The site title
+In the `<head>` section, Replace the text inside the title tag to the title you want. In my case, I am replacing it with 'Website Mockup Generator'
+
+```html 
+<title>Website Mockup Generator</title>
+```
+
+#### Changing the body background color and alignment
+In order to change the background color and the text alignment inside the body, specify a class called 'body' in the body tag. We will be providing CSS styling for the body class later.
+
+```html
+<body class="body">
+```
+
+#### The heading
+In the `<body>` provide the heading inside `<h1>...</h1>` tags. In my case, I am providig the heading as 'Website Mockup Generator'.
+
+```html
+<h1>Website Mockup Generator</h1>
+```
+
+#### Placing a text input field
+In order to place a text input field to get the website from the URL, use the following line to create a text input with a placeholder 'Enter the website' and provide id to the input as 'websiteAddress'.
+
+```html
+<input type="text" id="websiteAdress" placeholder='Enter the website'>

```

Comment:
typo in the id name

Reply:
um that was not a typo actually. it was intentional nvm, i will correct the id name